packages/horizon-householder-appeal-publish/src/config.js

```diff
@@ -11,4 +11,7 @@ module.exports = {
   openfaas: {
     gatewayUrl: process.env.GATEWAY_URL,
   },
+  logger: {
+    level: process.env.LOGGER_LEVEL || 'info',
```

Comment:
does a corresponding entry need adding to the helm config ?

Reply:
I don't know, does it?

Reply:
As best I understand it, `process.env.LOGGER_LEVEL` will be `undefined` when deployed beyond local dev. This will fall back to `info`, so all good in terms of valid config.

However, if you want or need to have a different log level beyond local dev then you would need to set up that config inside helm. 

I guess it's YAGNI, right?

Reply:
In that case, leave it. Ta.